sql/2021/compression/compression_format_trend.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+  # compression_format_trend.sql : What compression formats are being used (gzip, brotli, etc)
+SELECT
+  EXTRACT(YEAR FROM date) AS year,
+  client,
+  IF(resp_content_encoding = "gzip", "Gzip", IF(resp_content_encoding = "br", "Brotli", IF(resp_content_encoding = "", "no text compression", "other"))) AS compression_type,
+  COUNT(0) AS num_requests
+FROM
+  (
+    SELECT date, client, resp_content_encoding FROM `httparchive.almanac.requests_2020` WHERE DATE IN ('2020-08-01', '2019-07-01')
```

Comment:
Oh I just learned that `almanac.requests` now only includes 2021 data. I had intended to grow the same `almanac` tables with each year, but I think I understand why you did it this way. Does it have to do with the new fields and incompatible schema? If so I think it should work to add the 2019/2020 data back into `requests` with NULL values for those new fields.

The way it's organized now means that all of the previous years' queries that depend on this table will be broken, so it'd be good to fix it. I think the `summary_response_bodies` table has the same issue. Could you combine the respective tables so that 2019-2021 are all in one and update this query?

Reply:
We added a few new columns to the requests dataset for this years almanac, so I added them to a new table.  It probably makes sense to rerun the query for generating the requests table with the 2019 and 2020 datasets so we have them all in one place.

Reply:
I updated the requests table to include the 2019-2020 data and updated this query to remove the JOIN in the inner query.